pkgs/development/tools/misc/linuxkit/default.nix

```diff
@@ -2,28 +2,29 @@
 
 buildGoPackage rec {
   name = "linuxkit-${version}";
-  version = "2017-07-08";
-  rev = "8ca19a84d5281b1b15c7a48c21e5786943b47f1c";
+  version = "v0.6";
+  rev = "10f07ca1624102de3c2d93da7792be5528c32022";
 
   goPackagePath = "github.com/linuxkit/linuxkit";
 
   src = fetchFromGitHub {
     owner = "linuxkit";
     repo = "linuxkit";
     inherit rev;
-    sha256 = "150y7hnjhi81iik7np27y5466ldaackq72mpi7vmybbl7vr1wgw4";
+    sha256 = "12nph1sxgp7l2sb3ar7x8a2rrk2bqphca6snwbcqaqln2ixsh78i";
   };
 
   subPackages = [ "src/cmd/linuxkit" ];
 
   preBuild = ''
-    buildFlagsArray+=("-ldflags" "-X main.GitCommit=${builtins.substring 0 7 rev} -X main.Version=0.0.0")
+    buildFlagsArray+=("-ldflags" "-X main.GitCommit=${builtins.substring 0 7 rev} -X main.Version=0.6.0")
```

Comment:
It would be nice to convert the version into this main.Version attribute, so there is less to update each time. Can it just be `main.Verison=${version}` if you get the 'v' out of `version`?

Reply:
I just tried using `nix-review` on this PR and I see linuxkit doesn't know what version it is:

```
[nix-shell:~/.cache/nix-review/pr-52059]$ ./results/linuxkit/bin/linuxkit version
linuxkit version unknown
commit: unknown
```